src/code/z_player_lib.c

```diff
@@ -1,153 +1,1021 @@
+
+void func_80127B64(struct_801F58B0 arg0[], UNK_TYPE arg1, Vec3f* arg2);
+
+s32 func_801226E0(GlobalContext* globalCtx, s32 arg1) {
+    if (arg1 == 0) {
+        func_80169E6C(globalCtx, 0, 0xBFF);
+        if (globalCtx->sceneNum == SCENE_KAKUSIANA) {
+            return 1;
+        }
+    }
+
+    gSaveContext.respawn[0].data = 0;
+    return arg1;
+}
+
+s32 func_80122744(GlobalContext* globalCtx, struct_80122744_arg1* arg1, u32 arg2, Vec3s* arg3) {
+    arg1->unk_00 = arg2;
+    arg1->unk_01 = 0;
+    arg1->unk_04 = arg3;
+    return 1;
+}
+
+s32 func_80122760(GlobalContext* globalCtx, struct_80122744_arg1* arg1, f32 arg2) {
+    if (arg1->unk_01 < arg1->unk_00) {
+        Player* player = GET_PLAYER(globalCtx);
+        Vec3f sp30;
+        s32 pad;
+        s16 yaw;
+        f32 distXZ;
+
+        Math_Vec3s_ToVec3f(&sp30, &arg1->unk_04[arg1->unk_01]);
+        yaw = Math_Vec3f_Yaw(&player->actor.world.pos, &sp30);
+        func_800B6F20(globalCtx, &globalCtx->actorCtx.unk_26C, arg2, yaw);
+        globalCtx->actorCtx.unk268 = 1;
+        distXZ = Math_Vec3f_DistXZ(&player->actor.world.pos, &sp30);
+
+        if ((fabsf(player->actor.world.pos.y - sp30.y) < 50.0f) && (distXZ < arg2)) {
+            arg1->unk_01++;
+        }
+
+        return 0;
+    }
+
+    return 1;
+}
+
+void func_80122868(GlobalContext* globalCtx, Player* player) {
+    OPEN_DISPS(globalCtx->state.gfxCtx);
+
+    if (player->invincibilityTimer > 0) {
+        s32 phi_v0 = 50 - player->invincibilityTimer;
+
+        phi_v0 = CLAMP(phi_v0, 8, 40);
+
+        player->unk_B5F += phi_v0;
+        POLY_OPA_DISP =
+            Gfx_SetFog(POLY_OPA_DISP, 255, 0, 0, 0, 0, 4000 - (s32)(Math_CosS((player->unk_B5F << 8)) * 2000.0f));
+    } else if (gSaveContext.unk_1016 != 0) {
+        player->unk_B5F += 10;
+        POLY_OPA_DISP =
+            Gfx_SetFog(POLY_OPA_DISP, 0, 0, 255, 0, 0, 4000 - (s32)(Math_CosS((player->unk_B5F << 8)) * 2000.0f));
+    }
+
+    CLOSE_DISPS(globalCtx->state.gfxCtx);
+}
+
+void func_801229A0(GlobalContext* globalCtx, Player* player) {
+    OPEN_DISPS(globalCtx->state.gfxCtx);
+
+    if ((gSaveContext.unk_1016 != 0) || (player->invincibilityTimer > 0)) {
+        POLY_OPA_DISP = func_801660B8(globalCtx, POLY_OPA_DISP);
+    }
+
+    CLOSE_DISPS(globalCtx->state.gfxCtx);
+}
+
+void func_801229EC(UNK_TYPE arg0, UNK_TYPE arg1) {
+}
+
+extern s16 D_801BFDA0[PLAYER_MASK_MAX - 1];
+#if 0
+s16 D_801BFDA0[PLAYER_MASK_MAX - 1] = {
+    OBJECT_MASK_TRUTH,  OBJECT_MASK_KERFAY,  OBJECT_MASK_YOFUKASI, OBJECT_MASK_RABIT,   OBJECT_MASK_KI_TAN,
+    OBJECT_MASK_JSON,   OBJECT_MASK_ROMERNY, OBJECT_MASK_ZACHO,    OBJECT_MASK_POSTHAT, OBJECT_MASK_MEOTO,
+    OBJECT_MASK_BIGELF, OBJECT_MASK_GIBUDO,  OBJECT_MASK_GERO,     OBJECT_MASK_DANCER,  OBJECT_MASK_SKJ,
+    OBJECT_MASK_STONE,  OBJECT_MASK_BREE,    OBJECT_MASK_BAKURETU, OBJECT_MASK_BU_SAN,  OBJECT_MASK_KYOJIN,
+    OBJECT_MASK_BOY,    OBJECT_MASK_GORON,   OBJECT_MASK_ZORA,     OBJECT_MASK_NUTS,
+};
+#endif
```

Comment:
What's the motivation for externing this? IIRC, this was the data that was incorrectly split into `z_parameter.c`. Is there a reason that fix isn't included in this PR?
EDIT: Coming back to this, so none of the data is imported yet?

Reply:
Yeah, data isn't imported yet to minimize the size of the PR, you can take a look at how much data there is in the linked branch.
I added this as an `#if 0` so people can know what this specific variable contains